src/vzlogger.cpp

```diff
@@ -105,6 +105,39 @@ const char *long_options_descs[] = {
 	NULL /* stop condition for iterator */
 };
 
+void openLogfile() {
+	FILE *logfd = fopen(options.log().c_str(), "a");
+
+	if (logfd == NULL) {
+		print(log_alert, "opening logfile \"%s\" failed: %s", (char *)0, options.log().c_str(),
+			  strerror(errno));
+		return EXIT_FAILURE;
```

Comment:
this doesn't really exit now. (doesn't the compiler complain here?)

Reply:
i changed that to `exit(EXIT_FAILURE)`.
that's not perfect, but i'm afraid anything beyond that exceeds my level of caring about this corner case.

Reply:
should be sufficient. thx.